container/lxd/manager.go

```diff
@@ -237,7 +237,10 @@ func (m *containerManager) getContainerSpec(
 		Profiles: instanceConfig.Profiles,
 		Devices:  nics,
 	}
-	spec.ApplyConstraints(m.server.serverVersion, cons)
+	err = spec.ApplyConstraints(m.server.serverVersion, cons)
```

Comment:
We really, really need to turn on errcheck. I wonder how many issues this could cause in the wild?